src/parquet/column/properties.h

```diff
@@ -78,24 +86,38 @@ class PARQUET_EXPORT ReaderProperties {
 ReaderProperties PARQUET_EXPORT default_reader_properties();
 
 static int64_t DEFAULT_PAGE_SIZE = 64 * 1024 * 1024;
+static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
 static int64_t DEFAULT_DICTIONARY_PAGE_SIZE = DEFAULT_PAGE_SIZE;
-static Encoding::type DEFAULT_ENCODING = Encoding::PLAIN;
+static int64_t DEFAULT_DICTIONARY_SIZE_THRESHOLD = DEFAULT_PAGE_SIZE;
+static constexpr Encoding::type DEFAULT_DICTIONARY_FALLBACK_ENCODING = Encoding::PLAIN;
+static constexpr Encoding::type DEFAULT_VALUE_ENCODING = Encoding::PLAIN_DICTIONARY;
+static constexpr Encoding::type DEFAULT_DICTIONARY_ENCODING = Encoding::PLAIN_DICTIONARY;
```

Comment:
I see the `IS_DICTIONARY_ENABLED` above, I would suggest striking this altogether